packages/@o3r/localization/schematics/add-localization-key/schema.json

```
@@ -34,6 +34,11 @@
     "updateTemplate": {
       "type": "boolean",
       "description": "Update the template by replacing matching value by the localization key"
+    },
+    "keyToKebabCase": {
+    "type": "boolean",
+    "description": "Convert the key to kebab-case",
+    "default": false
```

```suggestion
      "type": "boolean",
      "description": "Convert the key to kebab-case",
      "default": false
```
small indentation issue